tests/python/UnitTestUtils.py

```diff
@@ -0,0 +1,123 @@
+import os
+from random import randint
+
+
+SIMPLE_CONFIG = """ocio_profile_version: 1
+
+search_path: luts
+strictparsing: false
+luma: [0.2126, 0.7152, 0.0722]
+
+roles:
+  default: raw
+  scene_linear: lnh
+
+displays:
+  sRGB:
+    - !<View> {name: Film1D, colorspace: vd8}
+    - !<View> {name: Raw, colorspace: raw}
+
+active_displays: []
+active_views: []
+
+colorspaces:
+  - !<ColorSpace>
+    name: raw
+    family: raw
+    equalitygroup: ""
+    bitdepth: 32f
+    description: |
+      A raw color space. Conversions to and from this space are no-ops.
+
+    isdata: true
+    allocation: uniform
+
+  - !<ColorSpace>
+    name: lnh
+    family: ln
+    equalitygroup: ""
+    bitdepth: 16f
+    description: |
+      The show reference space. This is a sensor referred linear
+      representation of the scene with primaries that correspond to
+      scanned film. 0.18 in this space corresponds to a properly
+      exposed 18% grey card.
+
+    isdata: false
+    allocation: lg2
+
+  - !<ColorSpace>
+    name: vd8
+    family: vd8
+    equalitygroup: ""
+    bitdepth: 8ui
+    description: |
+      how many transforms can we use?
+
+    isdata: false
+    allocation: uniform
+    to_reference: !<GroupTransform>
+      children:
+        - !<ExponentTransform> {value: [2.2, 2.2, 2.2, 1]}
+        - !<MatrixTransform> {matrix: [1, 2, 3, 4, 0, 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1], offset: [1, 2, 0, 0]}
+        - !<CDLTransform> {slope: [0.9, 1, 1], offset: [0.1, 0.3, 0.4], power: [1.1, 1.1, 1.1], sat: 0.9}
+"""
+
+SIMPLE_CDL = """
+<ColorCorrection id='shot 042'>
```

Comment:
Can you elaborate on this? It probably makes sense to use cdl files that exist in $[CMAKE_BINARY_DIR]/tests/data/files/, correct?

Reply:
I just meant that `SIMPLE_CDL` could be defined the the test module for CDLTransform, rather than UnitTestUtils.py, to keep that test more self contained. Currently the values in `SIMPLE_CDL` are defined in this definition and in the CDL test, and they must match. I thought that would be more clear if they lived in the same place.